pyinterface/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+cmake_policy(VERSION 3.14)
```

```suggestion
cmake_minimum_required(VERSION 3.14...3.31)
```

Unless you are explicitly trying to enforce CMAke `v3.14`'s policies specifically, I'd suggest letting CMake try to use more modern versions if available.